Bindings/bindings.json

```diff
@@ -1474,6 +1474,14 @@
                     ]
                 },
                 {
+                    "name": "apiKey",
+                    "value": "string",
+                    "defaultValue": "SendGridApiKey",
```

Comment:
I did think about that, but ultimately I based this on the Twilio setup, which gives shorter setting name like this.  I was trying to pick a short, easy-to-remember value here to optimize for people manually entering this into App Settings since this won't have a picker. I didn't see any bindings that have default values of the `AzureWebJobs*` variety anywhere else, either -- granted a lot of those have pickers, though, so it doesn't apply.